scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-8a.yaml

```diff
@@ -111,7 +111,6 @@ proposals:
       backend_name: nfs
       nfs:
         nfs_shares: ##cinder-storage-shares##
-        nfs_mount_options: nfsvers=3,rw,sync,nofail
```

Comment:
Why are those removed? Are they not needed anymore? Could you add a comment to the commit message?

Reply:
The nfs options were removed from cinder barclamp config in https://github.com/crowbar/crowbar-openstack/pull/972 and crowbar complains about having them there. Will update the commit message.

Reply:
Thanks! That means though that you won't have those NFS options for the cinder mounts. If they are necessary you need to add them somehow to the `nfs_shares` key one line above.

Reply:
The mounts work well without them for the nfs server we use for testing, seems they are in fact not necessary. If we would need them later, I would rather put them into the job parameter to bring them via `sed` to every scenario during build without having to duplicate the values for each scenario file.

Reply:
Thanks, that makes sense.